src/main.rs

```diff
@@ -154,10 +158,12 @@ struct Args {
 }
 
 /// The cache should get updated if this was requested by the user, or if auto
-/// updates are enabled and the cache age is longer than the auto update interval.
+/// updates are enabled and the cache age is longer than the auto update interval,
+/// unless it is explicitly disabled by the user.
```

Comment:
Code is fine, I wonder if we want to trim this comment a bit though:

> The cache should be updated if it was explicitly requested, or if an automatic update is due and allowed.

@dbrgn We usually disagree about comments :sweat_smile: , what do you think about this one?

Reply:
Your suggestion looks good.

Reply:
Excellent way to word this.